core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -62,7 +62,7 @@ THE SOFTWARE.
 
       <f:bottomButtonBar>
         <f:submit value="${%Save}" />
-        <f:apply />
+        <f:apply value="${%Apply}"/>
```

Comment:
In order to be able to translate the Apply button. If no value is provided, 'Apply'is not translated.